package.json

```diff
@@ -110,7 +110,7 @@
     "@pollyjs/core": "^5.0.0",
     "@pollyjs/persister-fs": "^5.0.0",
     "@release-it/bumper": "^2.0.0",
-    "@saleor/app-bridge": "^0.1.9",
+    "@saleor/app-bridge": "github:saleor/app-bridge#e4b81c2",
```

Comment:
Is this temporary change? After app-bridge is merged & released, will you change here to `0.2.0`?

Reply:
Yes (I also wrote it in the PR's description)

Reply:
Ok, keeping comment open until its replaced :)

Reply:
app-bridge 0.1.10 was releases

https://github.com/saleor/app-bridge/releases/tag/v0.1.10